cms/templates/js/signatory-editor.underscore

```diff
@@ -0,0 +1,23 @@
+<% console.log("signatory-editor.underscore.start") %>
+     <div class="signatory-panel-default">
+       <a class="signatory-panel-delete" href="#" data-tooltip="Delete">
```

Comment:
Already discussed, but wanted to note here that "Delete" still needs to be implemented.